node/migrations/000001_initialize_tables.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS adapters (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    active BOOLEAN NOT NULL DEFAULT TRUE
+);
+
+CREATE TABLE IF NOT EXISTS feeds (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    definition JSONB NOT NULL,
+    adapter_id INT8 NOT NULL,
```

Comment:
`INT8` can give us at most 256 different adapters. We aim higher, let's use at least `INT16` 💪